### DISCUSSION TITLE
husky bad variable name

### DISCUSSION THREAD

**Discussion by shivarm** [Hyperfoil/Horreum#422] [General]
# During commit i got error
```
git commit -m "update routes"

```
.husky/pre-commit: 5: export: sharma/Desktop/Open-Source/Horreum/webapp/node/node:/usr/lib/git-core:/home/shivamsharma/.nvm/versions/node/v18.13.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/lib/wsl/lib:/mnt/c/Python311/Scripts/:/mnt/c/Python311/:/mnt/c/Program: bad variable name
husky - pre-commit hook exited with code 2 (error)

**Comment by jesperpedersen** [comment]
Is this on native Windows ?

**Reply by shivarm** [reply]
Yes but in wsl Ubuntu environment

**Comment by jesperpedersen** [comment]
I have disabled husky for now - it is part of #385 now
